wagtailio/services/templates/services/services_page.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base.html" %}
+
+{% load static wagtailcore_tags wagtailimages_tags %}
+
+{% block extra_css %}
+    <link rel="stylesheet" type="text/css" href="{% static 'css/vendor/fontawesome-free-5.15.3-web/css/all.min.css' %}" />
```

Comment:
question: what's this used for? is it something we can achieve using SVGs instead?

Reply:
Right. That's a left over before I started using the SVG sprite. I have removed this now.